tally_ho/apps/tally/templates/super_admin/quarantine_checks_list.html

```diff
@@ -20,6 +20,7 @@ <h1>{% trans 'Quarantine checks List' %}</h1>
         <th>{% trans 'Modification date' %}</th>
         <th>{% trans 'Value' %}</th>
         <th>{% trans 'Percentage' %}</th>
+        <th>{% trans 'Status' %}</th>
         <th>{% trans 'Actions' %}</th>
     </tr>
 {% for oneCheck in checks %}
```

Comment:
we can simply call this `check in checks`

Reply:
Done